src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountsReader.java

```diff
@@ -220,12 +220,10 @@ private ReadCountRecord extractRecordWithTargetNameExtractorAndTargetCollection(
      * @throws UserException.BadInput if the overlapped target's location is not exactly the one provided in the input.
      * @return {@code null} if there is no target at {@code where}.
      */
-    private <E> Target createRecordTarget(final TargetCollection<E> targets, final SimpleInterval where) {
-        final E target = targets.target(where);
-        if (target == null) {
-            return null;
-        } else if (targets.location(target).equals(where)) {
-            return new Target(targets.name(target), where);
+    private Target createRecordTarget(final TargetCollection<Target> targets, final SimpleInterval where) {
+        final Target target = targets.target(where);
+        if (target == null || targets.location(target).equals(where)) {
```

Comment:
This is not the same.  The old code created a copy, which is implied by a method called ``create....`` and might be good idea to keep here.